docs/docs/reference/other-new-features/explicit-nulls.md

```diff
@@ -0,0 +1,296 @@
+---
+layout: doc-page
+title: "Explicit Nulls"
+---
+
+This proposal describes a modification to the Scala type system that makes reference types
```

Comment:
proposal -> Explicit nulls is an opt-in feature that modifies the Scala type system...